addons/stock/models/stock_orderpoint.py

```diff
@@ -335,82 +335,64 @@ def _get_orderpoint_action(self):
         orderpoints_removed = orderpoints._unlink_processed_orderpoints()
         orderpoints = orderpoints - orderpoints_removed
         to_refill = defaultdict(float)
-        all_product_ids = []
```

Comment:
Could you also add a test with a location that is not the main one inside the warehouse ? Thanks